src/main/java/com/playmonumenta/libraryofsouls/LibraryOfSouls.java

```diff
@@ -113,6 +125,9 @@ public void onDisable() {
 	}
 
 	public static LibraryOfSouls getInstance() {
+		if (INSTANCE == null) {
+			throw new RuntimeException("LibraryOfSouls not loaded");
```

Comment:
Nice way of solving this!

Reply:
Agreed; I've used this elsewhere to kill off "might be null" warnings.

Reply:
I think I stole this idea from someone else (possibly NickNackGus). Does work well, and doesn't make the caller's life unnecessarily complicated.